package.json

```diff
@@ -30,7 +30,7 @@
     "lint": "turbo run lint",
     "lint:fix": "turbo run lint --force",
     "prettier": "prettier \"**/*.{js,mjs,ts,tsx,md,mdx,json,yml,css}\" --check --cache --cache-strategy=content --cache-location=.prettiercache",
-    "prettier:fix": "npm run prettier --write",
+    "prettier:fix": "npm run prettier -- --write",
```

Comment:
Why N

Reply:
I tried without it and it didn't write. I'm happy to remove it but I think it was broken